### DISCUSSION TITLE
更新winget上的小狼毫到v0.15.0

### DISCUSSION THREAD

**Discussion by hensuu** [rime/home#1378] [General]
目前winget上的小狼毫輸入法仍然停留在v0.14.3，希望能夠更新到最新版v0.15.0

**Comment by mirtlecn** [comment]


**Reply by fxliang** [reply]
关于静默首次部署，`WeaselDeployer.exe /deploy` 就可以实现第一次安装静默部署，应该修改下nsis脚本就可以的，PR well come
